data_steward/tools/add_cdr_metadata.py

```diff
@@ -162,7 +207,11 @@ def add_metadata(dataset_id, project_id, table_fields, field_values=None):
             args.rdr_source, args.rdr_export_date, args.cdr_generation_date,
             args.qa_handoff_date, args.vocabulary_version
         ]))
-    print(field_values_dict)
+
```

Comment:
nit: move `field_values_dict` into `if args.component == INSERT:` because it's only used there.